bundles/automation/org.eclipse.smarthome.automation.module.core/src/main/java/org/eclipse/smarthome/automation/module/core/handler/ChannelPayloadTriggerHandler.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 1997, 2015 by ProSyst Software GmbH and others.
```

Comment:
Right, that is why I asked why you didn't change it to "by Deutsche Telekom AG and others". It should be the same as e.g. https://github.com/eclipse/smarthome/blob/b3149975fc65dfad6dbd74bf35478bb7f9a786a5/bundles/automation/org.eclipse.smarthome.automation.module.media/src/main/java/org/eclipse/smarthome/automation/module/media/internal/MediaScriptScopeProvider.java#L2.